master_xml.py

```diff
@@ -40,8 +40,8 @@ def getRawTooltips(locale):
             continue
         tooltipId = split[1].replace("tooltip_","").replace("_description","").replace("\"", "").lstrip("0")
 
-        # Remove any quotation marks, new lines and html tags.
-        rawTooltips[tooltipId] = cleanHtml(split[2].replace("\"", "").replace("\\n", "\n"))
+        # Remove any quotation marks and new lines.
+        rawTooltips[tooltipId] = split[2].replace("\"", "").replace("\\n", "\n")
```

Comment:
The `"` replacement is not actually an issue.

The line in the raw file looks like this:
`"tooltips";"tooltip_0030_description";"<b><keyword=deploy>Deploy:</keyword></b> <keyword=spawn>Spawn</keyword> a Unicorn or a Chironex."`

And our output now looks like this:
`"en-US": "<b><keyword=deploy>Deploy:</keyword></b> <keyword=spawn>Spawn</keyword> a Unicorn or a Chironex.\n"`

There is one change that needs to be made to that replace call though, it needs to remove the trailing `\n`. Fixing that now.